extensions/amp-story/1.0/amp-story.js

```diff
@@ -1064,10 +1067,51 @@ export class AmpStory extends AMP.BaseElement {
       });
     }
 
+    this.buildScreenReaderBackButton_();
+
     // Will resolve when all pages are built.
     return storyLayoutPromise;
   }
 
+  /**
+   * Creates a focusable button for screen readers to navigate back.
+   * @private
+   */
+  buildScreenReaderBackButton_() {
+    const backButton = document.createElement('button');
+
+    const label = this.localizationService_.getLocalizedString(
+      LocalizedStringId.AMP_STORY_PAGINATION_BUTTON_PREVIOUS_PAGE_LABEL
+    );
+
+    backButton.classList.add('i-amphtml-story-screen-reader-back-button');
+    label && backButton.setAttribute('aria-label', label);
+    this.mutateElement(() => {
+      this.element.appendChild(backButton);
+    });
+
+    backButton.addEventListener('click', () => {
+      this.storeService_.dispatch(
```

Comment:
I think this event will be dispatched by the `amp-story-page-advancement.js` here, but I would double check (it's right after we check `this.shouldHandleEvent_()`, which is why I noticed this)

https://github.com/ampproject/amphtml/blob/60c34f6742e1f9aca148a2a23b9c82825e0b2833/extensions/amp-story/1.0/page-advancement.js#L656-L659